### DISCUSSION TITLE
Sunflower-land expansion water animation

### DISCUSSION THREAD

**Discussion by paluras** [sunflower-land/sunflower-land#1463] [Art]
Different animation for water movement .

https://user-images.githubusercontent.com/98059798/193306397-381a931e-9aaa-46e4-a9fc-2383e521e986.mp4

![V3](https://user-images.githubusercontent.com/98059798/193307183-243e7236-6a6e-4e9b-9013-cdfeee8a6595.gif)

https://user-images.githubusercontent.com/98059798/193306403-d4e19790-8846-435a-b128-7b30c6643948.mp4

![V4](https://user-images.githubusercontent.com/98059798/193307063-9146fc0d-af88-4911-affb-8f1721706bd5.gif)

https://user-images.githubusercontent.com/98059798/193306406-d99813d7-20f1-44e9-b902-c75bf3694856.mp4

![v1](https://user-images.githubusercontent.com/98059798/193306932-0b3fbfd5-31af-4e76-b2f5-e9aa2275bd42.gif)

https://user-images.githubusercontent.com/98059798/193306410-47b54391-ac02-4158-a819-6985033baa6b.mp4

![v2](https://user-images.githubusercontent.com/98059798/193307225-e0450e46-2fc7-406e-953a-f0298aba3a9e.gif)

**Comment by adamhannigan** [comment]
I noticed this in Community Island and was amazed ❤️ These are incredible.

We will implement these across our different islands. I think we will lean towards one that is less active - we don't want to draw too much attention from the mainland, but we want to make it look alive 💪

**Reply by paluras** [reply]
Glad you like it ❤️ .
If it needs any tweaking let me know.
